com.unity.netcode.gameobjects/Tests/Editor/Timing/NetworkTimeTests.cs

```diff
@@ -9,6 +9,54 @@ namespace Unity.Netcode.EditorTests
 {
     public class NetworkTimeTests
     {
+        [Test]
+        [TestCase(0d, 0)]
+        [TestCase(5d, 0)]
+        [TestCase(-5d, 0)]
+        [TestCase(0d, -20)]
+        [TestCase(5d, int.MinValue)]
+        [TestCase(-5d, -1)]
+        public void TestFailCreateInvalidTime(double time, int tickrate)
+        {
+            Assert.Throws<UnityEngine.Assertions.AssertionException>(() => new NetworkTime(tickrate, time));
+        }
```

Comment:
are we saying, they should always be positive?

Reply:
Time values can be negative. With the idea that you can add a negative time to a positive time etc. But tickrate must always be positive. A 0 or negative tick rate does not make sense.

Reply:
would you like to comment this or something similar explaining "why these values are invalid?" inline? :)